00-Introducing-HOOMD-blue/01-The-Simulation-Object.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<p>For a new user it might be helpful to say what metric is best to rank the efficiency of the simulation (e.g., TPS) and where they can find this metric (e.g., in a logfile, in the standard out).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I've reworded this slightly. HOOMD v3 doesn't write out any information to stdout or logging by default - these will be the subject of a later tutorial.</p>